### DISCUSSION TITLE
exception has occured

### DISCUSSION THREAD

**Discussion by RussoNich** [Monika-After-Story/MonikaModDev#10319] [QA - tech support + general]
Hello i'd like to know how do i fix this exception everytime i open.

![Screenshot 2024-10-12 145615](https://github.com/user-attachments/assets/03d30aed-6a0c-41bb-866b-a5c4a76fc92e)
![Screenshot 2024-10-12 145606](https://github.com/user-attachments/assets/fce6ad9e-6c63-45c9-94b3-9236be7ba03c)

**Comment by Booplicate** [comment]
Make a new install of MAS
